Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to goerli
+   a) support restarting of nodes
+3. Improvements to Management contract
+   a) Support withdrawals
+4. Mempool improvements
+   a) Remove failed transactions
```

Comment:
Are these causing a user annoyance at the moment, or do they just get flushed out on restarts but in the meantime go unnoticed?

Reply:
they appear in logs, and are generally annoying